jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/app/pollNewTopic.js

```diff
@@ -0,0 +1,41 @@
+$(document).ready(function() {
+
+    if ($("div.pollItemsValue").length == maxPollItems) {
+        $("a#add").hide();
+    }
+
+    $( "#sortable" ).sortable({update: function(event, ui) {reIndex()}});
+
+    var name = $("#sortable").attr("rel");
+
+    $("a#add").click(function(e){
+        e.preventDefault();
+        length = $("div.pollItemsValue").length;
+        if(length < maxPollItems) {
+            newItem = $(".pollItemsValue:last").clone();
+            $(newItem).insertAfter(".pollItemsValue:last");
+            $("input[type=text]", newItem).val("");
+            reIndex();
+            if ($("div.pollItemsValue").length == maxPollItems) {
+                $("a#add").hide();
+            }
+        }
+    });
+
+    $(".pollItemsValue .remove").live("click", function(e){
+        e.preventDefault();
+        if($("div.pollItemsValue").length > minPollItems) {
+            $(this).closest('div.pollItemsValue').remove();
+            reIndex();
+            if ($("div.pollItemsValue").length < maxPollItems) {
```

Comment:
i think this check is unnecessary. if removed, then the length would be always less than maxPollItems